lib/double_entry/line_metadata.rb

```diff
@@ -1,5 +1,18 @@
 module DoubleEntry
   class LineMetadata < ActiveRecord::Base
+    class SymbolWrapper
+      def self.load(string)
+        return unless string
+        string.to_sym
+      end
+
+      def self.dump(symbol)
+        return unless symbol
+        symbol.to_s
+      end
+    end
+
     belongs_to :line
+    serialize :key, SymbolWrapper
```

Comment:
interesting idea! ive typically seen the getter specified so that it casts it to a symbol (the setter will automatically convert it to a string when it needs to be written)